plugins/emmc/README.md

```diff
@@ -31,6 +31,16 @@ One deprecated instance ID is also added; new firmware should not use this.
 The firmware is deployed when the device is in normal runtime mode, but it is
 only activated when the device is rebooted.
 
+## Quirk Use
+
+This plugin uses the following plugin-specific quirks:
+
+### EmmcBlockSize
```

Comment:
This is a slightly off-topic, and I will do a more proper review of this one tomorrow

I think we want a similar quirk for UFS.
Lately we encountered a UFS component, that can only be updated with a one-shot writing of entire blob.
We expect either vendor or OEM to make a PR for ufs-utils shortly, and the timing of this fwupd CL could not be better :)

Reply:
Thanks.  Will take it offline with Keren.

Reply:
Well, you now how it is: there is a spec and there is a reality of deviation all across the board :D in any case, that was just an offtopic comment - I will prob get to that particular part myself when I am back from my trips